build/cmake/functions.cmake

```diff
@@ -422,42 +432,35 @@ macro(wx_add_library name)
             set(runtime_dir "bin")
         endif()
         wx_install(TARGETS ${name}
+            EXPORT ${name}Targets
```

Comment:
I think the easier way is export all the targets in one group files such as `wxWidgetsTargets.cmake`, so we don't need to include all the targets files in `wxWidgetsConfig.cmake`.

Reply:
Yes, I'll do this.